src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaTemplateContent.java

```diff
@@ -105,11 +106,40 @@ public String getSchedule3AppliesParagraph(List<Descriptor> descriptors) {
+        }
+        return "";
+    }
+
+    private String getTelephoneOrVideoHearingTypeSentence(String hearingType, String appellantName, String bundlePage, boolean appellantAttended, boolean presentingOfifficerAttened) {
+        if ((appellantAttended && presentingOfifficerAttened) || (appellantAttended && !presentingOfifficerAttened)) {
+            return "This has been a remote hearing in the form of a " + hearingType + " hearing. " + appellantName + " attended the hearing today and gave oral evidence which was considered by the Tribunal together with the appeal bundle to page " + bundlePage + ".  " + (presentingOfifficerAttened ? "A" : "No") + " Presenting Officer attended on behalf of the Respondent.";
+        } else if ((!appellantAttended && presentingOfifficerAttened) || (!appellantAttended && !presentingOfifficerAttened)) {
+            return "This has been a remote hearing in the form of a " + hearingType + " hearing. " + appellantName + " did not attend the hearing today. " + (presentingOfifficerAttened ? "A" : "No") + " Presenting Officer attended on behalf of the Respondent.\n"
+                    + "\n"
+                    + "Having considered the appeal bundle to page " + bundlePage + " and the requirements of rules 2 and 31 of The Tribunal Procedure (First-tier Tribunal)(Social Entitlement Chamber) Rules 2008 the Tribunal is satisfied that reasonable steps were taken to notify " + appellantName + " of the hearing and that it is in the interests of justice to proceed today.  ";
+        }
+        return "";
+    }
+
+    public String getFaceToFaceHearingTypeSentence(String appellantName, String bundlePage, boolean appellantAttended, boolean presentingOfifficerAttened) {
+        if ((appellantAttended && presentingOfifficerAttened) || (appellantAttended && !presentingOfifficerAttened)) {
+            return "This has been an oral (face to face) hearing. "
+                    + appellantName + " attended the hearing today and the tribunal considered the appeal bundle to page " + bundlePage
+                    + ". " + (presentingOfifficerAttened ? "A" : "No") + " Presenting Officer attended on behalf of the Respondent.";
+        } else if (!appellantAttended && presentingOfifficerAttened || !appellantAttended && !presentingOfifficerAttened) {
+            return appellantName + " requested an oral hearing but did not attend today. "
+                    + (presentingOfifficerAttened ? "A " : "No ")
+                    + "Presenting Officer attended on behalf of the Respondent. "
+                    + "\n" + "Having considered the appeal bundle to page " + bundlePage + " and the requirements of rules 2 and 31 of The Tribunal Procedure (First-tier Tribunal)(Social Entitlement Chamber) Rules 2008 the Tribunal is satisfied that reasonable steps were taken to notify Felix Sydney of the hearing and that it is in the interests of justice to proceed today. ";
```

Comment:
Assuming you don't want to hardcode this to Felix Sydney? The content here looks very similar (Or pretty much identical in parts) to the getTelephoneOrVideoHearingTypeSentence method above - is there any chance it can be combined to avoid duplication?